codalab_service.py

```diff
@@ -321,9 +336,20 @@ def has_callable_default(self):
 for worker_manager_type in ['cpu', 'gpu']:
```

Comment:
Why do we have different args for the default # of CPUs / memory based on the worker manager type?

Reply:
Users can use different types of VMs based on different types of worker manager. In our case (the CodaLab Team), we use a pool of VM's that have 6 CPU's and 55GB of memory for each VM for the gpu WorkerManager and another pool of VM's that have 4 CPU's and 13GB of memory for each VM for the cpu WorkerManager